en/db-migrations.md

```diff
@@ -119,6 +119,34 @@ $migration::run([
 ]);
 ```
 
+## Migration methods
+
+Each migration is an separate class that works as an entity for specific database table.
+Inside each class there are different methods that can occur during migration running.
+
+Each migration file (and class) can implement specific methods, that will be executed based on the operation requested.
+There are no restrictions on the logic encapsulated in each method.
+
+Here are tables with Migration Class methods. Table is sorted by order of execution: from earliest to latest.
```

Comment:
The tables below show the Migration Class methods. They are stored by order of execution, earliest to latest.